### PR TITLE
Levi/docs

### PR DESCRIPTION
**Pull request by levigroker**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ thus:
 
 The `package` query parameter value must represent a valid JSON string. Please ensure values in the JSON are properly escaped and the JSON object passes linting checks (see [https://jsonlint.com](https://jsonlint.com)).
 
-Additionally, because the value of the package query parameter contains JSON, it must be URL encoded such that the whole URL is valid. URL encoding, or “[percent encoding](https://en.wikipedia.org/wiki/Percent-encoding)” is a common practice and there are many resources available online to elaborate on the concept and techniques involved.
+Additionally, because the value of the package query parameter contains JSON, it must be URL encoded such that the whole URL is valid. URL encoding, or "[percent encoding](https://en.wikipedia.org/wiki/Percent-encoding)” is a common practice and there are many resources available online to elaborate on the concept and techniques involved.
```
Comment (nickbenedetti) [inline README.md]: If you're trying to use only non-angled quotes, wouldn't you want to adjust the second one too?

File: README.md [hunk 2]
```diff
@@ -85,13 +85,19 @@ logtenprox://v2/method?package={"metadata":{"application":"My Application", "ver
 
 ## Identifying Entities
 
-LogTen Pro utilizes the `flight_key` attribute on the `Flight` entity to uniquely identify flights provided from an external source. The `flight_key` attribute is a string value and is required to be unique for a given logbook. When a flight record is sent through the API that includes a `flight_key`, LogTen Pro will first attempt to locate a matching flight with that `flight_key`. If a create or modify operation is being run and a matching `Flight` already exists with that `flight_key`, that `Flight` will be modified with the provided data. If a matching `Flight` is not found, a new one will be created with the provided data. For removal operations, only flights with the matching `flight_key` will be removed. Please note: If the matching `Flight` is locked (`flight_isLocked` is set to `1`), the `Flight` will not be modified or deleted. 
+**IMPORTANT:** each entity *MUST* contain the `entity_name` attribute and specify the entity name.
+
+| Entity Names          |
```
Comment (nickbenedetti) [inline README.md:90]: I'd expect this to be singular `Entity Name`

**Review by nickbenedetti — APPROVED** [review]
I like breaking out the JSON. The spelling/grammar adjustments are trivial.

File: README.md (hunk 2 above)
Comment (levigroker) [reply README.md:90]: I think it's okay as a header.

File: README.md (hunk 1 above)
Comment (levigroker) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (levigroker) [reply README.md:90]: In the context of the table header, I think this makes sense.

**Pull request merged**